README.md

```diff
@@ -23,3 +24,41 @@
 
 
 # DroneVis: Full compatible drone library to automate computer vision algorithms on parrot drones.
+
+**DroneVis** is a cutting-edge drone software library that has been specifically designed for use with the AR. Drone 2.0. It has been extensively tested both indoors and outdoors, and offers a wide range of features including adaptability in connecting to the drone, advanced computer vision algorithms, and a user-friendly interface. This makes it easy for users to take full advantage of the drone's capabilities and control it with simple commands.All of the implemented real-time data, inference, and detection achieve a minimum ``fps >= 4.5`` on an Intel core 8 CPU.
```

Comment:
space needed after This makes it easy for users to take full advantage of the drone's capabilities and control it with simple commands.